ansible/create-solaris-esxi.md

```diff
@@ -0,0 +1,88 @@
+# Instructions for creating Solaris machines in ESXi
+
+## Creating the new machine template
+
+1. Login to https://esxi.adoptopenjdk.net. @gdams and @sxa have credentials if needed.
+1. Click `Create / Register VM`.
+1. Click `Create a new virtual machine`.
+1. Select a name and guest OS:
+    - Name: Provide a hostname
+    - Compatibility: `ESXi 6.5 virtual machine`.
+    - Guest OS family: `Other`.
+    - Guest OS version: `Oracle Solaris 10 (64-bit)`
+1. Leave the storage as `datastore1`.
+1. Customize settings:
+    - CPU: Select `4` unless more is needed.
+    - Memory: Select `8GB` unless more is needed.
+    - Hard disk 1: Select a minimum of 120GB.
+    - Network Adapter 1: Ensure that `VM Network` is selected and the `Connect` box is ticked.
+    - CD/DVD Drive 1: Select `Datastore ISO file`, then select `sol-10-u11-ga-x86-dvd.iso`.
+1. Click `Finish`
+
+
+## First boot and Solaris Installation
+
+### System Identification
+
+1. Once the machine has been created, click on it and select the `Power on` option.
+1. You should then be able to click the console screen in the left corner which should display the `GNU GRUB` loader.
+1. `Oracle Solaris` should boot by default or you can hit the enter button.
+1. Type `1` and then enter (Oracle Solaris Interactive default)
+1. Select the keyboard layout (in my case `UK-English`) and then hit `F2`.
+1. Press Enter in the screen test shell.
```

Comment:
Not sure what "Screen test shell" is but it may be worth pointing out that if the GUI interfact comes up then it will eventually time out and go back to a command line instal which is likely less hassle :-)